docs/api-reference/test-utils/browser-driver.md

```diff
@@ -6,115 +6,108 @@
 
-### newPage
+Launch a new browser instance.
 
-Opens a new tab in the browser
+Parameters:
 
-`driver.newPage({url, width, height})`
+* `options` (Object) - options to pass to [puppeteer.launch](https://github.com/GoogleChrome/puppeteer/blob/v1.11.0/docs/api.md#puppeteerlaunchoptions).
 
-* `url` = `http://localhost:8080` - The url to load in the page.
-* `width` = `1550` - The width of the page
-* `height` = `850` - The height of the page
-Returns a `Promise` that resolves when the page is open.
+Returns a `Promise` that resolves when the browser has started.
 
+### openPage(options)
 
-### exposeFunction
+Open a new tab in the browser. Only works after a browser instance is started:
 
-Exposes a function on the `window` object in the controlled browser and waits until the controlled script has called that function. In addition, a string parameter to the function will be returned (as the value of the resolved `Promise`).
+```js
+browserDriver.startBrowser().openPage({url: 'http://localhost'});
+```
 
-`driver.exposeFunction(functionToExpose)`
+Parameters:
 
-* `functionToExpose`=`sendMessage` (`String`)
-Returns: a `Promise` that resolves when the browser code calls the function to the string the browser script passed to the exposed function.
+* `options` (Object)
+  - `url` (String) - The url to load in the page. Default `http://localhost`.
+  - `exposeFunctions` (Object) - keys are function names to be added to the page's `window` object, and the values are callback functions to execute in Node.js. See [exposeFunction](https://github.com/GoogleChrome/puppeteer/blob/v1.11.0/docs/api.md#pageexposefunctionname-puppeteerfunction) for details.
+  - `onLoad` (Function) - callback when the page is loaded
+  - `onConsole` (Function) - callback when the page logs to console
+  - `onError` (Function) - callback when the page crashes
 
+Returns a `Promise` that resolves when the page is open.
 
-### stopBrowser
 
-Stops and closes the browser.
+### stopBrowser()
 
-`driver.stopBrowser()`
+Terminate the browser instance.
 
 Returns a `Promise` that resolves when the browser is closed.
 
 
-### setShellStatus
-
-`driver.setShellStatus(success)`
-
-Set the return value that will be visible to the shell, truthy values will generate 0 which represents success.
-
-
-### startServer
+### startServer(config)
 
 Runs a server in a new child process, that the browser can connect to.
 
-`driver.startServer({process, parameters, options})`
-
-* `process`=`'./node_modules/.bin/webpack-dev-server'` - The process to fork
-* `parameters`=`['--config', 'test/render/webpack.config.js']` - The parameters to the process
-* `options`=`{maxBuffer: 5000 * 1024}`
-
-Example:
 ```js
 driver.startServer({
-  process: './node_modules/.bin/webpack-dev-server',
-  parameters: ['--config', 'test/render/webpack.config.js'],
-  options: {maxBuffer: 5000 * 1024}
+  command: './node_modules/.bin/webpack-dev-server',
+  arguments: ['--config', 'test/webpack.config.js'],
```

Comment:
Nit: I personally think we should move `test/webpack.config.js` to the root folder in our repos. That is unrelated to this text of course, though I see no reason to emphasize `test` subdir here?